atlasdb-client/src/test/java/com/palantir/atlasdb/schema/cleanup/NullCleanupMetadataTest.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2018 Palantir Technologies, Inc. All rights reserved.
+ *
+ * Licensed under the BSD-3 License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://opensource.org/licenses/BSD-3-Clause
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.atlasdb.schema.cleanup;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+import org.junit.Test;
+
+import com.google.protobuf.InvalidProtocolBufferException;
+
+public class NullCleanupMetadataTest {
```

Comment:
You've written the same few tests twice. I think this is OK - presumably the `Null` and `Arbitrary` cases will soon be different.

Reply:
Interesting. They'll definitely diverge in tests for the relevant visitors, but in terms of these tests (which mostly center around serialization and object-equality) I'm a bit less certain that they will in the short run.

@tboam and I have talked about possibly implementing the queue for the `Arbitrary` case at a later date (where we'll need to know where to write the cells to be cleaned-up to), but that's not immediately planned!